morpheus/stages/general/trigger_stage.py

```diff
@@ -70,7 +70,7 @@ def node_fn(obs: mrc.Observable, sub: mrc.Subscriber):
 
             obs.pipe(ops.to_list(), ops.flatten()).subscribe(sub)
 
-        node = builder.make_node_full(self.unique_name, node_fn)
+        node = builder.make_node(self.unique_name, ops.build(node_fn))
```

Comment:
```suggestion
        node = builder.make_node(self.unique_name, ops.to_list(), ops.flatten())
```

This is functionally equivalent. Lets update to minimize using `ops.build()`